content/pages/tutorial/hypothesis-testing.rst

```diff
@@ -21,7 +21,7 @@ Tutorial - Hypothesis Testing in Python
 (English)
 
 This tutorial will guide attendees to conduct hypothesis testing in Python with
-real datasets, and help them use it in daily work.
+real datasets, and help attendees to use it in daily work.
```

Comment:
Thanks.  Could you remove the “to” after “attendees”?  Grammatically there shouldn’t be “to” after “help”.

Reply:
Sure, but you may want to read [Help somebody (to) do – Cambridge Dictionary](https://dictionary.cambridge.org/zht/%E8%AA%9E%E6%B3%95/%E8%8B%B1%E5%BC%8F%E8%AA%9E%E6%B3%95/help-somebody-to-do).

Reply:
That's British English.  Here I am following the American English convention.